INSTALL.md

```diff
@@ -0,0 +1,92 @@
+#Installation for Lnd and Btcd
+
+### If Glide isn't installed, install it:
+```
+$ go get -u github.com/Masterminds/glide
+```
+
+###Install lnd:
+```
+$ cd $GOPATH
+$ git clone https://github.com/lightningnetwork/lnd $GOPATH/src/github.com/lightningnetwork/lnd
+$ cd lightningnetwork/lnd
+$ glide install
+$ go install . ./cmd/...
+```
+
+###Create lnd.conf:
+**On MacOS, located at:**
+/Users/[username]/Library/Application Support/Lnd/lnd.conf
+
+**On Linux, located at:**
+~/.lnd/lnd.conf
+
+**lnd.conf:**
+```
+[Application Options]
+rpcuser=kek
+rpcpass=kek
+btcdhost=127.0.0.1
+debuglevel=trace
+```
+
+###Install btcutil: (must be from roasbeef fork, not from btcsuite)
+```
+$ go get -u github.com/roasbeef/btcutil
+```
+
+###Install btcd: (must be from roasbeef fork, not from btcsuite)
+```
+$ cd $GOPATH/src/github.com/roasbeef/btcd
+$ glide install
+$ go install . ./cmd/...
+```
+
+###Start btcd (will create rpc.cert and default btcd.conf):
+```
+$ btcd --testnet
+```
+(Note: It may take several minutes to find segwit-enabled peers.)
+
+###Add a limited username and password to btcd.conf and restart
+(Note: Replace `kek` with the username and password you prefer.)
+
+**On Linux:**
+```
+$ sed -i 's#; rpclimituser=whatever_limited_username_you_want#rpclimituser=kek#' ~/.btcd/btcd.conf
+$ sed -i 's#; rpclimitpass=#rpclimitpass=kek#' ~/.btcd/btcd.conf
+```
+
+**On MacOS:**
+```
+$ sed -i 's#; rpclimituser=whatever_limited_username_you_want#rpclimituser=kek#' /Users/[username]/Library/Application Support/Btcd/btcd.conf
+$ sed -i 's#; rpclimitpass=#rpclimitpass=kek#' /Users/[username]/Library/Application Support/Btcd/btcd.conf
+```
+
+**Then, on both:**
```

Comment:
The instructions that follow are for both MacOS and Linux. How about "Then, regardless of OS:" instead?